rebuild/test/unit/aggregateSignatures.test.ts

```diff
@@ -0,0 +1,24 @@
+import {expect} from "chai";
+import {aggregateSignatures, Signature} from "../../lib";
+import {makeNapiTestSets} from "../utils";
+
+describe("Aggregate Signatures", () => {
+  const sets = makeNapiTestSets(10);
+  const signatures = sets.map(({signature}) => signature);
```

Comment:
similar comments as in aggregatePublicKeys

Reply:
Updated.